### PR TITLE
Remove non-Trame Jupyter backends

### PR DESCRIPTION
**Pull request by tkoyama010**
### Overview

trame

This PR removes the deprecated Jupyter backend. They will all be replaced by [trame](https://kitware.github.io/trame/index.html) .

 

### Details

- https://github.com/pyvista/pyvista/issues/3690

### CODE REVIEW

File: tutorial/07_sphinx/README.rst [hunk 1]
```diff
@@ -115,32 +115,32 @@ Which is rendered as:
 
 Using the ``Panel`` backend with PyVista
```
Comment (tkoyama010) [inline tutorial/07_sphinx/README.rst:116]:
```suggestion
Using the ``Trame`` backend with PyVista
```

File: doc/source/getting-started.rst [hunk 2]
```diff
@@ -68,59 +68,31 @@ the following packages:
...
 
    .. tab:: conda
 
       .. code::
 
-         conda install -c conda-forge jupyterlab pythreejs ipyvtklink panel
+         conda install -c conda-forge jupyterlab trame
 
 
 You can then plot using Jupyterlab or Jupyter Notebook interactively with one of three backends:
```
Comment (akaszynski) [inline doc/source/getting-started.rst:80]: We should update this line and add a Trame example.

File: tutorial/01_basic/a_lesson_basic.py [hunk 3]
```diff
@@ -10,10 +10,10 @@
 module and external files.
 """
 
-# Configure for panel
+# Configure for trame
 import pyvista
 
-pyvista.set_jupyter_backend('panel')
+pyvista.set_jupyter_backend('trame')
```
Comment (akaszynski) [inline tutorial/01_basic/a_lesson_basic.py:16]: Shouldn't these all be default?

Comment (tkoyama010) [reply tutorial/01_basic/a_lesson_basic.py:16]:
```suggestion
```

File: tutorial/01_basic/README.rst [hunk 4]
```diff
@@ -32,9 +32,9 @@ Here's a very basic dataset you can download.
 .. jupyter-execute::
    :hide-code:
 
-   # Configure for panel
+   # Configure for trame
    import pyvista
-   pyvista.set_jupyter_backend('panel')
+   pyvista.set_jupyter_backend('trame')
```
Comment (tkoyama010) [inline tutorial/01_basic/README.rst:37]:
```suggestion
```

File: tutorial/01_basic/README.rst [hunk 5]
```diff
@@ -32,9 +32,9 @@ Here's a very basic dataset you can download.
 .. jupyter-execute::
    :hide-code:
 
-   # Configure for panel
+   # Configure for trame
```
Comment (tkoyama010) [inline tutorial/01_basic/README.rst:35]:
```suggestion
```

File: tutorial/02_mesh/README.rst [hunk 6]
```diff
@@ -53,9 +53,9 @@ You can create one by defining a 2D array of Cartesian coordinates like so:
 .. jupyter-execute::
    :hide-code:
 
-   # Configure for panel
+   # Configure for trame
```
Comment (tkoyama010) [inline tutorial/02_mesh/README.rst:56]:
```suggestion
```

File: tutorial/02_mesh/README.rst [hunk 7]
```diff
@@ -53,9 +53,9 @@ You can create one by defining a 2D array of Cartesian coordinates like so:
...
    import pyvista
-   pyvista.set_jupyter_backend('panel')
+   pyvista.set_jupyter_backend('trame')
```
Comment (tkoyama010) [inline tutorial/02_mesh/README.rst:58]:
```suggestion
```

File: tutorial/03_figures/README.rst [hunk 8]
```diff
@@ -43,9 +43,9 @@ When plotting, users must first create a :class:`pyvista.Plotter` instance (much
 .. jupyter-execute::
    :hide-code:
 
-   # Configure for panel
+   # Configure for trame
```
Comment (tkoyama010) [inline tutorial/03_figures/README.rst:46]:
```suggestion
```

File: tutorial/03_figures/README.rst [hunk 9]
```diff
@@ -43,9 +43,9 @@ When plotting, users must first create a :class:`pyvista.Plotter` instance (much
...
    import pyvista
-   pyvista.set_jupyter_backend('panel')
+   pyvista.set_jupyter_backend('trame')
```
Comment (tkoyama010) [inline tutorial/03_figures/README.rst:48]:
```suggestion
```

File: tutorial/04_filters/README.rst [hunk 10]
```diff
@@ -41,9 +41,9 @@ object:
 .. jupyter-execute::
    :hide-code:
 
-   # Configure for panel
+   # Configure for trame
```
Comment (tkoyama010) [inline tutorial/04_filters/README.rst:44]:
```suggestion
```

File: tutorial/04_filters/README.rst [hunk 11]
```diff
@@ -41,9 +41,9 @@ object:
...
    import pyvista
-   pyvista.set_jupyter_backend('panel')
+   pyvista.set_jupyter_backend('trame')
```
Comment (tkoyama010) [inline tutorial/04_filters/README.rst:46]:
```suggestion
```

File: tutorial/06_vtk/README.rst [hunk 12]
```diff
@@ -254,9 +254,9 @@ example:
 .. jupyter-execute::
    :hide-code:
 
-   # Configure for panel
+   # Configure for trame
```
Comment (tkoyama010) [inline tutorial/06_vtk/README.rst:257]:
```suggestion
```

File: tutorial/06_vtk/README.rst [hunk 13]
```diff
@@ -254,9 +254,9 @@ example:
...
    import pyvista
-   pyvista.set_jupyter_backend('panel')
+   pyvista.set_jupyter_backend('trame')
```
Comment (tkoyama010) [inline tutorial/06_vtk/README.rst:259]:
```suggestion
```

File: tutorial/07_sphinx/README.rst [hunk 14]
```diff
@@ -186,15 +186,15 @@ Or you can first hide code that sets up the plotting backend and global theme::
...
        # specified in ``show``
-       pv.set_jupyter_backend('panel')
+       pv.set_jupyter_backend('trame')
 
 .. jupyter-execute::
    :hide-code:
 
    import pyvista as pv
-   pv.set_jupyter_backend('panel')
+   pv.set_jupyter_backend('trame')
```
Comment (tkoyama010) [inline tutorial/07_sphinx/README.rst:197]:
```suggestion
```

File: tutorial/07_sphinx/README.rst [hunk 15]
```diff
@@ -186,15 +186,15 @@ Or you can first hide code that sets up the plotting backend and global theme::
        import pyvista as pv
 
        # Set the global jupyterlab backend.  All plots from this point
-       # onward will use the ``panel`` backend and do not have to be
+       # onward will use the ``trame`` backend and do not have to be
        # specified in ``show``
-       pv.set_jupyter_backend('panel')
+       pv.set_jupyter_backend('trame')
```
Comment (tkoyama010) [inline tutorial/07_sphinx/README.rst:191]:
```suggestion
```

File: tutorial/07_sphinx/README.rst [hunk 16]
```diff
@@ -154,7 +154,7 @@ Jupyter notebooks.  You can use it on a plot by plot basis by setting the
        point_cloud = pv.PolyData(dataset.points[::100])
        point_cloud['height'] = point_cloud.points[:, 2]
        point_cloud.plot(window_size=[500, 500],
-                        jupyter_backend='panel',
+                        jupyter_backend='trame',
```
Comment (tkoyama010) [inline tutorial/07_sphinx/README.rst:157]:
```suggestion
```

File: tutorial/07_sphinx/README.rst [hunk 17]
```diff
@@ -171,7 +171,7 @@ And here's the resulting output in Sphinx:
     point_cloud = pv.PolyData(dataset.points[::100])
     point_cloud['height'] = point_cloud.points[:, 2]
     point_cloud.plot(window_size=[500, 500],
-                     jupyter_backend='panel',
+                     jupyter_backend='trame',
```
Comment (tkoyama010) [inline tutorial/07_sphinx/README.rst:174]:
```suggestion
```

**Pull request merged**